Tools/TestWebKitAPI/Tests/IPC/StreamConnectionTests.cpp

```diff
@@ -346,11 +346,11 @@ TEST_P(StreamMessageTest, SendWithSwitchingDestinationIDs)
     });
 
     for (uint64_t i = 0u; i < 777u; ++i) {
-        auto success = m_clientConnection->send(MockStreamTestMessage1 { }, defaultDestinationID(), defaultSendTimeout);
-        EXPECT_TRUE(success);
+        auto result = m_clientConnection->send(MockStreamTestMessage1 { }, defaultDestinationID(), defaultSendTimeout);
+        EXPECT_TRUE(result == IPC::ConnectionError::NoError);
```

Comment:
Typically these are
`EXPECT_EQ(result, IPC::ConnectionError::NoError);` but if you opt into the explicit bool, then I think it makes sense to preserve these unmodified.

Reply:
Fixed.